consensus/src/threshold_simplex/mocks/supervisor.rs

```diff
@@ -428,3 +428,72 @@ impl<P: PublicKey, V: Variant, D: Digest> Monitor for Supervisor<P, V, D> {
         (latest, rx)
     }
 }
+
+/// Passive is a thin wrapper around Supervisor that disables signing by returning
+/// None for share(). This is useful for tests that need to run a non-active observer
+/// that still receives and verifies consensus certificates.
+#[derive(Clone)]
+pub struct Passive<P: PublicKey, V: Variant, D: Digest> {
```

Comment:
I wonder if we could more cleanly support this by just accepting `None` in `Supervisor` for share?

I'm probably missing something...